src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.csproj

```diff
@@ -11,11 +11,29 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <OutputType>exe</OutputType>
     <X86ProjectDirectory>..\..\tools\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86\</X86ProjectDirectory>
+
+    <!--
+      This prevents dotnet from automatically generating a lib/tfm/*.dll; in turn we're then able
+      to exclude the lib/net461 and manually construct only the necessary folders:
+      - lib/netcoreapp2.0/...
+      - runtimes/win7-x64/lib/...
+      - runtimes/win7-x86/lib/...
+    -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="build\**\*" Pack="true" PackagePath="%(Identity)" />
-    <None Include="$(X86ProjectDirectory)\bin\$(Configuration)\net461\win7-x86\$(MSBuildThisFileName)-x86.exe" Pack="true" PackagePath="lib\net461\$(MSBuildThisFileName)-x86.exe" />
+    <None Include="build\common.targets" Pack="true" PackagePath="%(Identity)" />
+    <None Include="lib\net461\_._" Pack="true" PackagePath="%(Identity)" />
+    <None Include="build\netcoreapp2.0\*" Pack="true" PackagePath="%(Identity)" />
```

Comment:
> Also, could we merge L26 and this like before?

If I merged I'd also get the net461 folder over on Linux which would never be valid.



> Do you need to specify PackagePath?

Looks like I do. WIthout it you get a structure like this:

![image](https://user-images.githubusercontent.com/2008729/26846044-6cb45b84-4aad-11e7-8f21-1f3479063659.png)